event-statistics/src/test/java/io/quarkus/sample/superheroes/statistics/KafkaProducerResource.java

```diff
@@ -27,7 +34,8 @@
 
 	@Override
 	public Map<String, String> start() {
-		this.fightProducer = new KafkaProducer<>(producerConfig(), new StringSerializer(), new ObjectMapperSerializer<>());
+    RegistryClientFactory.setProvider(new VertxHttpClientProvider(Vertx.vertx()));
```

Comment:
You will need to close that Vert.x instance at some point.

Reply:
I just pushed a commit that closes these (in both `rest-fights` & `event-statistics`).